charms/__init__.py

```diff
@@ -0,0 +1,2 @@
+from pkgutil import extend_path
```

Comment:
I pulled this one from charms.reactive verbatim. My reading suggests it's needed for the charms. namespace to work.